pkg/translator/testdata/simple_grpc_hpsf.yaml

```diff
@@ -9,6 +9,8 @@ components:
       - name: Port
         value: 4317
         type: number
+  - name: ShowMeTheOlly
+    kind: OTelDebugExporter
```

Comment:
should this end up in a different test? its not just a grpc test anymore 😬

Reply:
It probably should, yep. I started here for convenience, but I'll pop it out into a separate test case.